sr_port/gbldefs.c

```diff
@@ -1190,3 +1191,33 @@ GBLDEF	char		io_setup_errstr[IO_SETUP_ERRSTR_ARRAYSIZE];
 GBLDEF	void		(*mupip_exit_fp)(int4 errnum);	/* Function pointer to mupip_exit() in MUPIP but points to a routine
 							 * that assert fails if run from non-MUPIP builds.
 							 */
+GBLDEF	CLI_ENTRY	*cmd_ary;	/* Pointer to command table for MUMPS/DSE/LKE etc. */
+
+/* GT.CM OMI related global variables */
+GBLDEF	bool		neterr_pending;
+GBLDEF	int4		omi_bsent = 0;
```

Comment:
Just a comment, the zero initializations aren't necessary. They are automatic. Not sure if the compiler is smart enough to ignore these or if it keeps a separate copy of initialized value for these that it sets when an image initializes.

Reply:
I verified that omi_bsent ended up in the uninitialized section. So the compiler is smart enough. I am therefore not making any changes. Hope that is okay.

> nm $gtm_dist/libgtmshr.so | grep omi_bsent
000000000063ea28 b omi_bsent

> man nm
           "b" The symbol is in the uninitialized data section (known as BSS).